MovieAppPetProject/MovieAppPetProject/HomeScreen/NewTableViewCell.swift

```diff
@@ -6,15 +6,14 @@
 //
 
 import UIKit
-import SDWebImage
 
 class NewTableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionViewDataSource {
-    
-//    private let nowPlayingViewModel = NowPlayingViewModel()
+
     var nowPlayingMovies: [NowPlayingResults] = []
```

Comment:
var nowPlayingMovies = [NowPlayingResults]() just looks cleaner. Don't have to change, just a suggestion😄